src/App.tsx

```diff
@@ -15,48 +16,102 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  SORT_BY_ALPHABET = 'SORT_BY_ALPHABET',
+  SORT_BY_LENGTH = 'SORT_BY_LENGTH',
```

Comment:
let's write enum keys with camel case or uppercase https://www.typescriptlang.org/docs/handbook/enums.html
```suggestion
  Alphabet = 'alphabet',
```